File: Python/1041 - Coordinates of a Point.py
x,y = map(float,input().split(" "))
if x == y and y == 0:
    print("Origem")
elif x != 0 and y == 0:
    print("Eixo X")
elif x == 0 and y != 0:
    print("Eixo Y")
else:
    if y > 0:
        if x > 0:
            print("Q1")
        else:
            print("Q2")
    else:
        if x > 0:
            print("Q4")
        else:
            print("Q3")

File: C++/1041 - Coordinates of a Point.cpp
#include <iostream>

std::string get_quadrant(double x,double y)
{
    if(x == 0)
    {
        return "Eixo Y";
    }
    if(y == 0)
    {
        return "Eixo X";
    }
    if (x > 0)
    {
        if (y > 0)
        {
            return "Q1";
        }
        else
        {
            return "Q4";
        }
    }
    else
    {
        if (y > 0)
        {
            return "Q2";
        }
        else
        {
            return "Q3";
        }
    }
}

int main()
{
    double x,y;
    std::cin >> x >> y;
    if (x == 0 && y == 0)
    {
        printf("Origem\n");
    }
    else
    {
        std::cout << get_quadrant(x,y) << std::endl;
    }
    return 0;
}